tools/monitoring/prometheus/prometheus.yml

```diff
@@ -4,4 +4,4 @@ scrape_configs:
       static_configs:
       - targets:
         # goshimmer prometheus plugin export
-        - 127.0.0.1:9311
\ No newline at end of file
+        - goshimmer:9311
```

Comment:
This prometheus config file is not exclusively for docker. If I wanted to run prometheus + grafana as standalone apps or system services, I can't rely on docker host name resolution.

When ran with docker, the goshimmer container should expose 9311 to the host network (127.0.0.1/0.0.0.0), and prometheus should listen to this.